File: Tools/Calculator/arithmeticCalculator.java
/*
A simple arithmetic calculator using java language

Silky1099
*/

import java.util.*;

public class arithmeticCalculator {
    public static void calc(int a, char op, int b) {
        switch (op) {
            case '+':
                System.out.println(a + b);
                return;
            case '-':
                System.out.println(a - b);
                return;
            case '*':
                System.out.println(a * b);
                return;
            case '/':
                System.out.println(a / b);
                return;
            default:
                System.out.println("Enter a valid operator!!!");
                main(null);
        }

    }

    public static void main(String[] args) {
        try (Scanner sc = new Scanner(System.in)) {
            System.out.println("Enter two numbers: ");
            int x = sc.nextInt(), y = sc.nextInt();
            System.out.println("Enter the operation to be performed: (+ - * /)");
            char op = sc.next().charAt(0);
            calc(x, op, y);
        }
    }
}

File: Tools/calculator/arithmetic_calculator.c
/*
This program can be used to perform the most basic arithmetic operations (addition, subtraction, multiplication, division) on two digits

by Silky1099
*/

#include<stdio.h>
#include<conio.h>
int main(){
    float a,b;
    char o;
    clrscr();
    printf("Enter the operator(+,-,*,/): ");
    scanf("%c",&o);
    printf("Enter the first and second numbers: ");
    scanf("%f %f",&a,&b);
    switch (o){
        case '+':
            printf("%f",a + b);
            break;
        case '-':
            printf("%f",a - b);
            break;
        case '*':
            printf("%f",a * b);
            break;
        case '/':
            printf("%f",a / b);
            break;
        default:
            printf("You entered a wrong operator!");
    }
    getch();
    return 0;
}
